cluster/stone-prd-rh01/managed/rhtap-releng-tenant/role-releng-gitops-repo-rw.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
tenants-config does not support Roles currently. See https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/resources/stonesoup/argocd-control-plane/rh-managed-workspaces-config-project.yml#L22

Reply:
Dropped in ce39299.